binding.gyp

```diff
@@ -24,7 +24,10 @@
       "./native/secp256k1/src",
       "<!(node -e \"require('nan')\")"
     ],
-    "defines": [],
+    "defines": [
+      "ECMULT_GEN_PREC_BITS=4",
+      "ECMULT_WINDOW_SIZE=15",
```

Comment:
Please explain these parameters and why they were chosen.

Reply:
based on this https://github.com/cryptocoinjs/secp256k1-node/blob/master/binding.gyp#L36-L39